vector/src/main/res/layout/fragment_timeline.xml

```diff
@@ -103,14 +95,13 @@
     <im.vector.app.core.ui.views.TypingMessageView
         android:id="@+id/typingMessageView"
         android:layout_width="0dp"
-        android:layout_height="20dp"
+        android:layout_height="wrap_content"
         android:paddingStart="20dp"
         android:paddingEnd="20dp"
-        app:layout_constraintBottom_toTopOf="@id/composerLayout"
+        app:layout_constraintBottom_toTopOf="@id/bottomBarrier"
```

Comment:
This view is now on top of the barrier, and the timeline is now on top of this view.